src/main/java/com/cloudbees/jenkins/support/impl/GCLogs.java

```diff
@@ -145,16 +147,36 @@ private void handleRotatedLogs(@Nonnull final String gcLogFileLocation, Containe
     @CheckForNull
     private String getGcLogFileLocation() {
 
-        String gcLogSwitch = vmArgumentFinder.findVmArgument(GCLOGS_JRE_SWITCH);
-        if (gcLogSwitch == null) {
-            LOGGER.fine("No GC Logging switch found, cannot collect gc logging files.");
-            return null;
+        String fileLocation;
+        
+        if(!isJava8OrBelow()) {
```

Comment:
I tend to prefer the main branches for the "positive" value, I think it's less confusing. WDYT?

Reply:
good point, will change this.